cdisc_rules_engine/models/failed_validation_entity.py

```diff
@@ -1,21 +1,27 @@
+from dataclasses import dataclass, field
 from cdisc_rules_engine.enums.execution_status import ExecutionStatus
-
 from .base_validation_entity import BaseValidationEntity
 
 
+@dataclass
 class FailedValidationEntity(BaseValidationEntity):
     """
     The entity describes an error that occurred during validation
     indicating that the process has finished its execution with error.
     """
 
+    dataset: str = field(init=False)
+    _error: str = field(init=False)
```

Comment:
Replace variables like these that have underscores with the actual variable name that we want to see in the as_dict representation.